src/OneTrustCookieConsentBanner.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC, useEffect } from 'react';
+
+type Props = {
+  domainScript: string;
+  enabled: boolean;
+};
+
+const OneTrustCookieConsentBanner: FC<Props> = ({ domainScript, enabled }) => {
```

Comment:
`www` uses `react-helmet`. Helmet does not allow you to pass "custom" elements as children, only native HTML. A component is considered as a custom element even if it only returns native elements. Another option would be to include react-helmet here in this library, but the library is quite massive for what it is doing so I opted not to pull it here.